content/docs/get-started/azure/begin.md

```diff
@@ -54,12 +54,28 @@ Next, we'll install the required language runtime.
 {{< install-dotnet >}}
 {{% /choosable %}}
 
-Next, we'll configure Azure.
+Finally, configure Pulumi with Microsoft Azure.
 
-### Configure Azure
+### Configure Pulumi to access your Microsoft Azure account
 
-<a href="{{< relref "/docs/intro/cloud-providers/azure/setup" >}}" target="_blank">Configure Azure</a> so the Pulumi CLI can connect to Azure. If you have previously configured the <a href="https://docs.microsoft.com/en-us/cli/azure/" target="_blank">Azure CLI</a>, `az`, Pulumi will respect and use your configuration settings.
+Pulumi requires cloud credentials to manage and provision resources. Pulumi can authenticate to Azure using an IAM user or service principal that has **Programmatic access** with rights to deploy and manage your Azure resources.
```

Comment:
It would be good to clarify that we don't access the credentials on our own servers. Authentication to Azure only happens locally.